admin/herbert_mex_mpi.m

```diff
@@ -5,7 +5,7 @@
 % Manually modify this script to specify the mpi libraries location in your
 % system.
 %
-use_her_mpich = true
+use_her_mpich =true;
```

Comment:
You have removed the balancing space to the right of the =. It would also be useful to fix the spelling of verbose from verbouse - always leaves the impression that perhaps a differently spelled instance may not be consistent.

Reply:
trailing space went off probably because I was switching it on/off and adding ; verbose done